homeassistant/components/ezviz/camera.py

```diff
@@ -32,166 +50,194 @@
     {
         vol.Required(CONF_USERNAME): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
-        vol.Optional(CONF_CAMERAS, default={}): {cv.string: CAMERA_SCHEMA},
+        vol.Optional(CONF_REGION, default=DEFAULT_REGION): cv.string,
+        vol.Optional(ATTR_CAMERAS, default={}): {cv.string: CAMERA_SCHEMA},
```

Comment:
Don't change this config schema. It's legacy and only used when importing the existing config to the config flow.

Reply:
Schema now matches original.